addons/hr_recruitment_survey/tests/test_recruitment_survey.py

```diff
@@ -60,11 +83,43 @@ def test_send_survey(self):
             set(answers.mapped('email')),
             set([self.job_sysadmin.email_from]))
 
+        # Tests ACL
+        # Manager: ok for survey type recruitment
+        with self.assertRaises(AccessError):
+            invite.with_user(self.hr_recruitment_manager).action_invite()
+        self.survey_sysadmin.survey_type = 'recruitment'
```

Comment:
I would argue that this is not a realistic flow (changing the type of a sent survey).

We could simply have several here, and so we'll have an `invite_recruitement_type` to ease understanding when reading.